src/Mailer/SmtpMailer.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer;
+
+use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddress;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Email;
+use Twig\Environment as TwigEnvironment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Error\SyntaxError;
+
+final class SmtpMailer implements Mailer
+{
+    private TwigEnvironment $twig;
+    private MailerInterface $mailer;
+    private LoggerInterface $logger;
+    private Address $from;
+    /** @var string[] */
+    private array $whiteListedDomains;
+
+    public function __construct(TwigEnvironment $twig, MailerInterface $mailer, LoggerInterface $logger, Address $from, array $whiteListedDomains)
+    {
+        $this->twig = $twig;
+        $this->mailer = $mailer;
+        $this->logger = $logger;
+        $this->from = $from;
+        $this->whiteListedDomains = $whiteListedDomains;
+    }
+
+    public function send(EmailAddress $to, string $subject, string $htmlTemplate, string $textTemplate, array $variables = []): bool
+    {
+        try {
+            $html = $this->twig->render($htmlTemplate, $variables);
+            $text = $this->twig->render($textTemplate, $variables);
+        } catch (LoaderError|SyntaxError|RuntimeError $e) {
+            $this->logger->error('[TwigTemplate] ' . $e->getMessage());
+            return false;
+        }
+
+        $email = (new Email())
+            ->from($this->from)
+            ->to($to->toString())
+            ->subject($subject)
+            ->text($text)
```

Comment:
I think the rendering of the txt template can be done automatically based on the html by symphony

Reply:
I just tested this but this seems not accurate, but maybe I am missing a detail?
- When sending the HTML template instead of the text the HTML is still in the plain text version.
- When not calling the text() method it does not sent a plain text version at all.

Reply:
Not sure if we want that. I think for links this would be a problem, because then the URL would be lost because it is inside the href property.